test/integration/framework/util.go

```diff
@@ -35,25 +34,9 @@ const (
 	currentPodInfraContainerImageVersion = "3.1"
 )
 
-// GetServerArchitecture fetches the architecture of the cluster's apiserver.
-func GetServerArchitecture(c clientset.Interface) string {
-	arch := ""
-	sVer, err := c.Discovery().ServerVersion()
-	if err != nil || sVer.Platform == "" {
-		// If we failed to get the server version for some reason, default to amd64.
-		arch = "amd64"
-	} else {
-		// Split the platform string into OS and Arch separately.
-		// The platform string may for example be "linux/amd64", "linux/arm" or "windows/amd64".
-		osArchArray := strings.Split(sVer.Platform, "/")
-		arch = osArchArray[1]
-	}
-	return arch
-}
-
-// GetPauseImageName fetches the pause image name for the same architecture as the apiserver.
+// GetPauseImageName returns the pause image name with proper version
 func GetPauseImageName(c clientset.Interface) string {
```

Comment:
minor not: his probably doesn't need to take `c` anymore.

Reply:
after removing that c this function is dup of function from manifest.go file, so I consolidated all the functions into one and refactored the code a bit. PTAL at latest commits.